### ISSUE TITLE
Installing from within the editor should work smoothly

### ISSUE THREAD

**Issue by mathetos** [impress-org/donation-block-for-stripe#31]
When you install the block from within the block editor you get a fata error message and can't use the block. 

Instead, when installing this way, the block should immediately be available. 
![install-donation-form-block](https://user-images.githubusercontent.com/930724/161808184-ace42605-0aa4-4543-b57d-8d39b500de31.gif)

**Comment by mathetos** [comment]
I enabled WP_DEBUG and got a slightly different error:
![image](https://user-images.githubusercontent.com/930724/161828551-80056016-5742-4482-8b92-1c620f0be033.png)

Then in debug.log I see two issues that are most likely related:
```
PHP Notice:  Trying to get property 'livePublishableKey' of non-object in C:\Users\reach\Local Sites\750words\app\public\wp-content\plugins\donation-block-for-stripe-by-givewp\src\DonationBlock.php on line 54
```

```
PHP Notice:  Trying to get property 'testPublishableKey' of non-object in C:\Users\reach\Local Sites\750words\app\public\wp-content\plugins\donation-block-for-stripe-by-givewp\src\DonationBlock.php on line 56
```

**Comment by DevinWalker** [comment]
Hmm, I actually fixed those noticed in the repo version. I'm assuming you're using WP.org to download, which does have the PHP notices but shouldn't be causing what you're seeing. If you have time let's hop on a call soon to review.

**Comment by DevinWalker** [comment]
I found this JS error occurs on fresh install from the WP.org plugin directory:
![2022-04-05_13-40-23](https://user-images.githubusercontent.com/1571635/161846128-02f6d6e6-4158-48eb-af7c-ebeb7f39523f.png)

I believe I've already fixed this, just need to test it out to make sure.

**Comment by DevinWalker** [comment]
I'm seeing a different error now that we've released `1.0.3`:
![2022-04-07_11-34-01](https://user-images.githubusercontent.com/1571635/162273176-09fc0d76-2a05-466b-8e62-542f8a793430.png)

Digging into the error more it points to:

![2022-04-07_11-34-17](https://user-images.githubusercontent.com/1571635/162273251-377c3cc5-c869-4367-b1b3-53ef342491e2.png)

Which I believe, in the uncompiled version, has to do with these lines: 
![2022-04-07_11-35-16](https://user-images.githubusercontent.com/1571635/162273323-e9fdc0de-73e4-4bbf-a019-b7ec94201124.png)

**Comment by DevinWalker** [comment]
Gutenberg releases since this was posted have fixed this issue.
